great_expectations/expectations/metrics/column_aggregate_metrics/column_partition.py

```diff
@@ -129,20 +137,25 @@ def _get_evaluation_dependencies(
         return dependencies
 
 
-def _get_column_partition_using_metrics(bins: int, n_bins: int, _metrics: dict) -> list:
+def _get_column_partition_using_metrics(
+    bins: int | Literal["uniform", "ntile", "quantile", "percentile", "auto"],
```

Comment:
Unless I'm missing something it doesn't seem like `bins` can ever be an `int`.

Reply:
I'll remove it and see if any new warning pop up.